ansible/roles/bigbluebutton/tasks/main.yml

```diff
@@ -33,12 +33,3 @@
   wait_for:
     timeout: 20
   delegate_to: localhost
-
-  # https://docs.bigbluebutton.org/greenlight/gl-admin.html#creating-an-administrator-account
-- name: Create Greenlight admin user
-  # https://yamllint.readthedocs.io/en/stable/disable_with_comments.html
-  # yamllint disable rule:line-length
-  command: >
-    docker exec greenlight-v2 bundle exec rake
-    user:create['{{ bbb_adm.name }}','{{ bbb_adm.email }}','{{ bbb_adm.pass }}','admin']
-  # yamllint enable rule:line-length
```

Comment:
Ok, we remove this from here and then add the user information to the variable `bbb_greenlight_users`. But that variable is never used in a task. So I think this means that we are not actually creating the default admin user, which I think is a bug.

Don't we need to add some task to create that for us? Or did I miss something?

Reply:
It's being used by the [upstream's ansible role](https://github.com/n0emis/ansible-role-bigbluebutton).
The variable is set in our inventory here: https://github.com/PrivacyLx/devops/pull/55/files#diff-1244efcc21eb365c7e47066a99ee7979c7e5eb3535ea5faab967d6b6b4cba215R69

Reply:
I see. Thanks for the clarification.